portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,12 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String quote = "beep boop - Ben Lin";
```

Comment:
nit: be consistent with indentation, in this case it might seem like this and line 33 are in different scopes, for example readers might think that this is in an if block if they don't read carefully

Reply:
I'll be sure to be consistent,